plugins/org/src/components/MyGroupsSidebarItem/MyGroupsSidebarItem.tsx

```diff
@@ -96,7 +97,7 @@ export const MyGroupsSidebarItem = (props: {
         {groups?.map(function groupsMap(group) {
           return (
             <SidebarSubmenuItem
-              title={group.metadata.title || group.metadata.name}
+              title={<EntityDisplayName entityRef={group} />}
```

Comment:
As we have some tests for `MyGroupsSidebarItem` already can we please add some more specifically for these cases just so we don't have regressions now and in the future, please? Ideally these should be new ones for the checks on `name`, `title`, and `displayName`.